packages/terra-application-name/tests/wdio/application-header-name-spec.js

```diff
@@ -1,7 +1,7 @@
 Terra.describeViewports('ApplicationHeaderName', ['tiny', 'medium'], () => {
   describe('Displays a default application header name', () => {
     before(() => browser.url('/#/raw/tests/terra-application-name/application-name/application-header-name-default'));
-    browser.pause(3000); // to display image
+    browser.waitForVisible('[class*="demo-image-container"]');
```

Comment:
So does we need to make similar changes for tabs also instead of adding browser.pause() : [here](https://github.com/cerner/terra-framework/blob/56487c47a6fc41f973fbb5bf817b6bd73e963c74/packages/terra-tabs/tests/wdio/tabs-spec.js#L12)

Reply:
In the "Application Header Name" scenario the icon was not being loaded/displayed by the time the screenshot was taken which caused the failure. So by using `browser.waitForVisible` we wait for the icon to be made visible before we proceed further.

Whereas, in the case of the "Responsive Tabs" issue, the Tabs are **already visible** and the failure occurs because of a mismatch in the width of the Tabs by 1 or 2 pixels. So even if we use `browser.waitForVisible` it wouldn't resolve the inconsistency.